gcloud-java-examples/README.md

```diff
@@ -89,6 +89,20 @@ To run examples from your command line:
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.storage.StorageExample" -Dexec.args="download <bucket_name> test.txt"
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.storage.StorageExample" -Dexec.args="delete <bucket_name> test.txt"
     ```
+ * Here's an example run of `Stat`.
```

Comment:
Nit: to make it slightly clearer what "Stat" is, you could say "Here's an example run of `Stat`, a sample that uses a GCS-based NIO file system"